ports/README.md

```diff
@@ -28,11 +28,8 @@ After copying the port, add an entry to the table below. You should also contrib
 
 ## List of port overlays
 
-| Port                       | Reason                                                                                                  |
-|----------------------------|---------------------------------------------------------------------------------------------------------|
-| `libmagic`                 | Updating to the upstream port deferred due to failures.                                                 |
-| `openssl`                  | Pinning to OpenSSL 1.1 until we can move to 3.0 in January 2024.                                        |
-| `pcre2`                    | To be removed alongside libmagic.                                                                       |
```

Comment:
The custom port for PCRE2 was applying some patches to its CMake build system, to make it usable from libmagic. These patches are now upstreamed.